In [1]:
import pandas as pd
import numpy as np
import scipy.stats as stats

**Step 1: import modules**

**Step 2: Create an array of values**

In [2]:
data = np.array([6,7,7,12,13,13,15,16,19,22])

**Step 3 : Calculate the z-score for each value in the array**

In [3]:
stats.zscore(data)

array([-1.39443338, -1.19522861, -1.19522861, -0.19920477,  0.        ,
        0.        ,  0.39840954,  0.5976143 ,  1.19522861,  1.79284291])

*Each z-score tells us how many standard deviations away an inviduals value is from the mean.For example:**
The first value"6" in the array is 1.394 standard below the mean.The fifth value of"13" in the array is 0 standard deviation away from the mean,i.e.it is equal to the mean . 
# The last value of"22" in the array is1.793standard deviation above the mean.

***Z-score normalization refers to the process of normalizing every value in a dataset such that the mean of all of the values is 0 and the standard deviation is 1.***